app/src/main/res/layout/quicksettings_website_info.xml

```diff
@@ -15,11 +15,21 @@
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="vertical"
+        android:orientation="horizontal"
         android:minHeight="@dimen/tracking_protection_item_height"
         android:paddingTop="8dp"
         android:paddingBottom="8dp">
 
+        <ImageView
+            android:id="@+id/favicon_image"
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:importantForAccessibility="no"
+            android:scaleType="fitCenter"
+            android:layout_marginStart="16dp"
+            android:layout_gravity="center_vertical"
+            tools:drawableStartCompat="@drawable/ic_internet" />
+
         <TextView
             android:id="@+id/url"
             style="@style/QuickSettingsText"
```

Comment:
We also will need to add `app:drawableEndCompat="@drawable/ic_arrowhead_right"` and `android:paddingEnd="0dp"` to add the arrow icon and align the icon with the **"Blocked items"**'s arrow section.

Reply:
I will add that

Reply:
See https://github.com/mozilla-mobile/fenix/pull/20144/commits/6ea5281dab40ec4762cbc3fbbe69af839d936efd#diff-e4bce07500a19f5ea4f0a09151a4ff7a0a5d823e6dfe4ae5cb3ecab6d4662524R66

Reply:
Ya that was intentional. I was just adding the favicon here.